test/fixtures.rb

```diff
@@ -339,7 +339,7 @@ class StoreLayout
       include Store::Helpers::AssetTagHelpers
 
       def head
-        %(<meta name="lotusrb-version" content="0.3.1">)
+        Lotus::Utils::Escape::SafeString.new %(<meta name="lotusrb-version" content="0.3.1">)
```

Comment:
@AlfonsoUceda I don't understand, this is only an example of a `<meta>` HTML tag. It isn't used by Lotus apps.

The fixture could've been something like this:

```ruby
Lotus::Utils::Escape::SafeString.new %(<meta name="foo" content="bar">)
```

:wink: